.github/workflows/emulator.yml

```diff
@@ -0,0 +1,134 @@
+name: emulator
```

Comment:
I would call this `test-android` or something like that but up to you

Reply:
Renamed the display name to Test Android, renamed the workflow file to test.yml, in case Test iOS might be possible sometime in the future

Reply:
still to do?

Reply:
I think GitHub tricked me - sorry for the noise